frappe/public/js/integrations/google_drive_picker.js

```diff
@@ -58,20 +65,34 @@ export default class GoogleDrivePicker {
 	createPicker() {
 		// Create and render a Picker object for searching images.
 		if (this.pickerApiLoaded && frappe.boot.user.google_drive_token) {
-			var view = new google.picker.DocsView(google.picker.ViewId.DOCS)
+			this.view = new google.picker.DocsView(google.picker.ViewId.DOCS)
 				.setParent('root') // show the root folder by default
 				.setIncludeFolders(true); // also show folders, not just files
 
-			var picker = new google.picker.PickerBuilder()
+			this.picker = new google.picker.PickerBuilder()
 				.setAppId(this.appId)
 				.setDeveloperKey(this.developerKey)
 				.setOAuthToken(frappe.boot.user.google_drive_token)
-				.addView(view)
+				.addView(this.view)
 				.setLocale(frappe.boot.lang)
 				.setCallback(this.pickerCallback)
 				.build();
 
-			picker.setVisible(true);
+			this.picker.setVisible(true);
+			this.setupHide();
+		}
+	}
+
+	setupHide() {
```

Comment:
Camel case 😢

---

ignoring for now... since the whole file needs to be refactored. (@barredterra can you take it up?)

Reply:
@surajshetty3416, do you want me to change all instances of camel case to snake case?

Reply:
Yes, can you take it up?